common/src/main/java/com/microsoft/identity/common/internal/platform/DevicePopManager.java

```diff
@@ -1044,7 +1080,7 @@ private KeyPair generateNewRsaKeyPair(@NonNull final Context context,
 
             // If the key material is hidden (HSM or otherwise) the length is -1
             if (length >= minKeySize || length < 0) {
-                logSecureHardwareState(kp);
+                getSecureHardwareState(kp);
```

Comment:
We don't need to capture the returned value here?

Reply:
Not here, no -- just using this method for its logging purposes

Reply:
Will add a comment to clarify this usage

Reply:
I think it is fine -- just a bit misleading when reading the code

Reply:
18db49fcb0166da6802345ce53ee7cb1679fd781